stan/math/rev/core/var.hpp

```diff
@@ -181,6 +184,56 @@ namespace stan {
       // NOLINTNEXTLINE
       var(unsigned long x) : vi_(new vari(static_cast<double>(x))) { }  // NOLINT
 
+      /**
+       * Construct a variable from the specified arithmetic argument
+       * by constructing a new <code>vari</code> with the argument
+       * cast to <code>double</code>, and a zero adjoint. Only works
+       * if the imaginary part is zero.
+       *
+       * @param x Value of the variable.
+       */
+      explicit var(std::complex<double> x) {
```

Comment:
Instances of `complex` should be passed by constant reference, `var(const std::complex<double>& x)`.

Reply:
This change results in a compiler error
```
error: cannot bind non-const lvalue reference of type ‘std::complex<long double>&’ to an rvalue of type ‘std::complex<long double>’
```

Reply:
Nevermind. I forgot the `const`.